js/script.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
```

Comment:
- [ ] **[OPTIONAL]** You did an excellent job working on your mobile menu! 💯 I loved the animation! However, It would be great if you prevent the user from scrolling the page when the menu is open, this will provide a better user experience and design! 🤓 🔧 

You can achieve this by, for example, also toggling a `noScroll` class in the body element that prevents this behavior:


```CSS
.noScroll {
  overflow: hidden;
}
``` 

```js
hamburger.addEventListener('click', () => {
  hamburger.classList.toggle('active');
  navMenu.classList.toggle('active');
  document.body.classList.toggle('noScroll');
});
```
Please consider implementing it! 🤖